test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/ClientTests/Tests/ClientOpenTypeUpdateTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="ClientOpenTypeUpdateTests.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.OData;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OData.Client.E2E.TestCommon;
+using Microsoft.OData.Client.E2E.Tests.ClientTests.Server;
+using Microsoft.OData.Client.E2E.Tests.Common.Clients.OpenTypes.Default;
+using Microsoft.OData.Client.E2E.Tests.Common.Server.OpenTypes;
+using Xunit;
+
+namespace Microsoft.OData.Client.E2E.Tests.ClientTests.Tests
+{
+    public class ClientOpenTypeUpdateTests : EndToEndTestBase<ClientOpenTypeUpdateTests.TestsStartup>
+    {
+        private readonly Uri _baseUri;

```

Comment:
Nitpik: Since you are not using this `_baseUri` in any other place other than the constructor, can you remove this here and move it inside the constructor.

You can do the same to the other TestsControllers

Reply:
I'm not sure there is any issue with this but I've removed them for now.